ci_scripts/propose_release.rb

```diff
@@ -45,13 +45,4 @@
 open_url jira_url
 notify_user
 
-rputs "Send an email to mobile-sdk-updates@stripe.com with the following information:"
```

Comment:
Hmm there's no reason to send out an email now?

Reply:
Yep, I set this up (in a somewhat Rube Goldberg-y way) to auto-forward from the GitHub Releases feed.